bin/spinnaker-monitoring.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+PYTHONWARNINGS=once \
+python "/opt/spinnaker-monitoring" \
+  --log_dir /var/log/spinnaker/monitoring monitor --datadog --period 15
```

Comment:
I'm going to add an external configuration file for the options.
So your install could write that file and introduce datadog (and perhaps period override) there. I think the default options will not monitor anything. They will only run the embedded webserver with some built-in operations. Maybe I'll add something more polling friendly. Then you explicitly override the options to configure to push somewhere.

I'll try to figure that out by Monday so I'll be ready for your debian packaging.